lib/Behat/Context/SelectionFieldTypeFormContext.php

```diff
@@ -75,7 +75,7 @@ public function theSelectFieldShouldBeFlaggedAsRequired()
                 'required',
                 $nodeElement->getAttribute('required'),
                 sprintf(
-                    'select with id %s is not flagged as required',
+                    'The select with ID %s is not flagged as required',
```

Comment:
what selected?

Reply:
This refers to a `select` element in a form